deploy/install/02-components/03-ui.yaml

```diff
@@ -38,10 +38,11 @@ metadata:
   name: longhorn-frontend
   namespace: longhorn-system
 spec:
+  type: ClusterIP
   selector:
     app: longhorn-ui
   ports:
-  - port: 80
-    targetPort: 8000
+  - name: http
+    port: 80
+    targetPort: http
```

Comment:
Target port changed?

Reply:
It points to this port https://github.com/longhorn/longhorn-manager/pull/579/files/f624488ee510fe9cf494208efd319a2f2998a43c#diff-5c41a8536687cf54789b966a37991635R26